src/fromager/sources.py

```diff
@@ -631,6 +640,11 @@ def build_sdist(
     return sdist_filename
 
 
+# meson dist currently only works with Git or Mercurial repos
+# affects NumPy and projects with https://mesonbuild.com/meson-python/
```

Comment:
If Meson doesn't work with an alternative VCS, it won't be used. I don't think Fromager should worry about this.

Reply:
I need to clarify this comment or remove `BUILD_BACKEND_FORCE_TARBALL` again. The `meson dist` command expects a VCS checkout. It does not support sdist -> sdist.